### PR TITLE
Update BestChange.com.fc

### PR DESCRIPTION
**Pull request by aviramdayan**
update-elements-and-time-options

### CODE REVIEW

File: BestChange.com/BestChange.com.fc [hunk 1]
```diff
@@ -462,36 +459,61 @@ public class firebitcoinfaucettech : FaucetScript
...
-				
-         Driver.Close();  // Close specific chrome window
-         Driver.Quit();  // also Close the console app, that was used to kick off the chrome window
-         Driver.Dispose();  // Close the chromedriver.exe
-		 //return GetWaitSetting();
-         return MinuteINT;
+         var TimeVar = ElementByXPath("//input[@id='bonus_button']");
+
+         int min,sec;
+         min = Convert.ToInt32(TimeVar.Text.Trim().Replace("Time left: ", "").Split('.')[0])*60;
```
Comment (faucetcollector) [inline BestChange.com/BestChange.com.fc]: I will keep asking to please use int.TryParse next time to improve script resilience

Comment (aviramdayan) [reply BestChange.com/BestChange.com.fc]: but this method `Int32`, in all my scripts work well all the time..

Comment (faucetcollector) [reply BestChange.com/BestChange.com.fc]: Yes, until something is changed and the value passed to ToInt32 isnt an int anymore then an Exception will be thrown and the script will fail.
So to improve resciliency its best to do int.TryParse so it doesnt throw an Exception but you can handle the case and return a custom time if it fails

Comment (aviramdayan) [reply BestChange.com/BestChange.com.fc]: I'll run more tests later, thanks for the info.

**Pull request merged**